script/install/debootstrap.bash

```diff
@@ -171,5 +171,48 @@ echo "$chroot_cmd update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9
 chroot "$ISOLATE_ROOT" update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 75
 # gcc 9 done
 
-# let user know that chroot installs are finished
+[ -z "$TRAVIS" ] && bash script/confirm.bash 'Install JavaScript V8 (submissions in V8 will fail without this!)' && {
+  # JavaScript V8 engine compile and install
+  echo "Preparing to compile and install the JavaScript V8 engine."
+  echo
+  echo "If this fails, visit https://v8.dev/docs/build and follow the instructions."
+  echo "This may take a while ..."
+  echo
+  set -x
+
+  : Making temporary directory
+  mkdir ~/v8-build
+  cd ~/v8-build
+
+  : Installing depot_tools
+  git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+  export PATH="$PATH:$PWD/depot_tools"
+  gclient
```

Comment:
```suggestion
  gclient --version # finishes the installation
```
This has the same effect, but doesn't print the long help message.
We could even remove this line completely; it's only required on Windows according to the [depot_tools tutorial](https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/depot_tools_tutorial.html#_setting_up), and on Linux the `fetch` command will also do the required work. However, I'd like to keep the line to match the instructions (at least loosely).

Reply:
I was also unsure if it's necessary but the instructions [here](https://v8.dev/docs/source-code) seemed to imply that it should be done on both Windows and Linux.